glossary

```diff
@@ -0,0 +1,36 @@
+# Appendix: Glossary of Terms
+
+**CTF**
+Compact C Type Format. Assists in debugging. See ctf(4)
+
+**CW**
+Compiler wrapper. Historical version of illumos were built with Sun Studio
+(see below). In order to transition to the `gcc` compiler, the compiler
+wrapper was developed to translate compiler flags from Sun Studio formatted
+`Makefile`s to `gcc` style compiler flags.
+
+**FMA**
+Fault management architecture.
+
+**Gate; illumos gate**
+Gate is old Sun Microsystems terminology for a software source-code repository.
+
+**Lint; linting**
+Using the `lint` tool to check for common programming errors.
+
+**ON**
+OS/Net (Operating system/Network) consolidation. In historical version of SunOS,
+OS and Net were separate software components. Now this term refers to the
+operating system as a whole. 
+
+**RTI**
+Request to Integrate. The final approval process needed for a change to become
+part of the illumos gate. See [Integrating Changes](./integrating.html)
+
+**Sun Studio**
+A compiler suite historically required to build illumos. As of 2016, dependency
+on Sun Studio has been eliminated with the exception of the `lint` tool.
```

Comment:
Technically, the default env files still shadow with Studio.  I'd love to stop that though!